### DISCUSSION TITLE
how do i initialise next 13 app in java script?

### DISCUSSION THREAD

**Discussion by avbeladiya** [vercel/next.js#42222] [Help] [answered]
I used this command to create new app 
```npx create-next-app@latest --experimental-app``` but it creates ts app by default, also tried with --javascript but it didn't work.

**Comment by icyJoseph** [comment, marked as answer]
It is an upcoming feature, that being said, you could just delete the tsconfig and all of the TS related stuff.

https://github.com/vercel/next.js/discussions/41745#discussioncomment-3985833

Reposting for visiblity:

```
> Just to be clear we're planning to rework create-next-app to give you 
> the option to choose between JavaScript or TypeScript so it'll solve this request.
> For `app` right now it'll stay TypeScript till that is implemented.
```

A bit controversial here perhaps, and making a huge assumptions, but you should consider learning TS as well, a new language never hurts and you don't have to unlearn JavaScript to do so, in the contrary, it'll help you out with your JS.

**Reply by avbeladiya** [reply]
thanks for the quick reply.

I'm working on TS with the previous version but starting with a new project, the entire team is not comfortable in TS yet, I was wondering if it was missed in doc or not available yet.

**Reply by icyJoseph** [reply]
Hi! They've released a new version of create-next-app which supports JS for the --experimental-app :) and in general the create-next-app experience is much nicer.

```
➜ yarn create next-app --experimental-app
yarn create v1.22.19
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 🔨  Building fresh packages...

success Installed "create-next-app@13.0.1" with binaries:
      - create-next-app
✔ What is your project named? … delete-me
✔ Would you like to use TypeScript with this project? … No / Yes
✔ Would you like to use ESLint with this project? … No / Yes
Creating a new Next.js app in /Users/user/projects/delete-me.

Using yarn.
```

And then I can see:

```
➜ ls app/
 globals.css
 layout.jsx
 page.jsx
 page.module.css
```
